Gulpfile.js

```diff
@@ -0,0 +1,87 @@
+var gulp = require('gulp');
+var gulpif = require('gulp-if');
+var uglify = require('gulp-uglify');
+var uglifycss = require('gulp-uglifycss');
+var concat = require('gulp-concat');
+var sass = require('gulp-sass');
+var sourcemaps = require('gulp-sourcemaps');
+var merge = require('merge-stream');
+var debug = require('gulp-debug');
+var livereload = require('gulp-livereload');
+
+var env = process.env.GULP_ENV;
+var rootPath = 'web/assets/';
+
+var paths = {
+    js: [
+        'node_modules/jquery/dist/jquery.min.js',
+        'node_modules/bootstrap/dist/js/bootstrap.min.js',
+        'node_modules/admin-lte/dist/js/app.min.js',
+    ],
+    'sass': [
+        'src/Sylius/Bundle/UiBundle/Resources/private/sass/**',
+    ],
+    css: [
```

Comment:
you should either use `css` or use `'css'` but your keys should use the same convention. Right now there are mix of two styles